pennylane_lightning/core/src/simulators/lightning_kokkos/StateVectorKokkos.hpp

```diff
@@ -112,6 +115,11 @@ class StateVectorKokkos final
         init_generators_indices_();
     };
 
+    /**
+     * @brief Get the total number of wires.
+     */
+    auto getTotalNumQubits() const -> size_t { return this->getNumQubits(); }
```

Comment:
Can we define that in the base class and simply override it in the MPI-supported backends?

Reply:
Good idea!